### PR TITLE
Update Apple TV documentation

### PR DESCRIPTION
**Pull request by postlund**
**Description:**
Update with instructions for new features.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#8122

### CODE REVIEW

**Comment by postlund** [comment]
Anyone reading this... What do I do to make rake generate new markdown files I have added (I have done ``rake generate`` and ``rake preview`` several times from a recently cloned repo, with no cached files)? I now have three files:

* apple_tv.markdown
* media_player.apple_tv.markdown
* remote.apple_tv.markdown

Only the media_player-file is generated, the other two are ignored. I get no error from rake whatsoever and changes I make to ``media_player.apple_tv.markdown`` update as expected. This is my general experience working with the home page and I'm starting to get a bit sad and tired because of how tiresome it is 😢

**Comment by Landrash** [comment]
@fabaff can you help with the question about rake?

**Comment by postlund** [comment]
I can add that I got it to generate correctly somehow by removing cached files, generating again several times and whatnot. No idea what actually "fixed" it. I had the exact same issue when writing documentation for the OTP platform and basically did the same thing until it eventually worked. Very, very annoying and if there is a solution I'm more than happy to know about it.

**Comment by postlund** [comment]
Can someone merge this and update the main page? 😊

**Comment by postlund** [comment]
Thanks @amelchio!

File: source/_components/media_player.apple_tv.markdown [hunk 1]
```diff
@@ -28,24 +28,102 @@ media_player:
...
+
+If you, when playing media with `play_url`, get the following error message:
+
+*“This AirPlay connection requires iOS 7.1 or later, OS X 10.10 or later, or iTunes 11.2 or later.”*
+
+then device authentication is required. Press the icon in the upper left corner and select the leftmost icon according to the image below:
+
+<img src='/images/screenshots/developer-tools.png' />
+
+Select `media_player` as domain, `apple_tv_authenticate` as service and enter `{'entity_id': 'XXX'}` into "Service Data", but replace XXX with the entity id of your device (e.g. `media_player.apple_tv`). Press the button and hopefully you are presented with an input dialog asking for a pin code:
```
Comment (mu3) [inline source/_components/media_player.apple_tv.markdown]: Select `apple_tv` as domain, `apple_tv_authenticate`

Comment (postlund) [inline source/_components/media_player.apple_tv.markdown]: I fixed that in another PR, thanks!

**Pull request merged**